test/common/testlib.py

```diff
@@ -1048,10 +1048,12 @@ def filterOutput(self, test, failed, output):
         cmd = [ "tests-policy", testvm.DEFAULT_IMAGE ]
         try:
             proc = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
-            (output, error) = proc.communicate(output)
+            (changed, unused) = proc.communicate(output)
```

Comment:
This might trigger some pyflake warning about the unused variable. For the future, just `communicate()[0]`. But fine for me for this quick-fix if semaphore doesn't croak.